src/analysis/retail/rogue/outlaw/modules/core/RestlessBlades.ts

```diff
@@ -41,7 +41,7 @@ class RestlessBlades extends Analyzer {
 
     let cdr = 1000;
     if (this.selectedCombatant.hasBuff(SPELLS.TRUE_BEARING.id)) {
-      cdr += 1000;
+      cdr += 500;
```

Comment:
Prefer using `const cdr = 1000 + (this.selectedCombatant.hasBuff(SPELLS.TRUE_BEARING.id) ? 500 : 0)` to be more concise. Also preferable to store these constants in global variables to make it easier to change things after tuning.

Reply:
What is the consensus regarding adding these type of global variables to the outlaw/constants.tsx file versus adding them directly into the RestlessBlades module?

Reply:
I wouldn't say theres a general consensus but usually if a constant is used in multiple files then use constants.tsx, otherwise then put it at top of file. Alternatively, just put every numerical constant in constants.tsx